.github/workflows/gh-pages.yml

```diff
@@ -65,3 +65,39 @@ jobs:
         with:
           deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
           publish_dir: ./public
+          
+  run-test-suite:
+    runs-on: ubuntu-18.04
+    timeout-minutes: 15
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Upgrade pip
+        run: python3 -m pip install --upgrade pip
+        
+      - name : Install pipenv
+        run : python -m pip install --upgrade pipenv wheel
```

Comment:
I think `wheel` is not needed here, or  can you explain why its necessary?

Reply:
Tbh I just copied and pasted a guide on running tests as part of a GitHub workflow and kept the commands as similar as possible. Removed `wheel` and everything seems to work fine.